src/Factory.php

```diff
@@ -141,13 +141,25 @@ public function target($name)
             if (preg_match('#^https?://#', $file)) {
                 $files[] = new Remote($file);
             } else {
-                $path = $scanner->find($file);
-                if ($path === false) {
-                    throw new RuntimeException("Could not locate $file for $name in any configured path.");
+                if (preg_match('/(.*)(\/\*.*?)$/', $file, $matches)) {
```

Comment:
Couldn't the glob expansion be done as separate case similar to how remote files are handled above.